docs/UserGuide.md

```diff
@@ -147,7 +147,7 @@ list-b
 Expected Output:
 
 ```
-Listed all bidders.
+Listed all bidder(s).
```

Comment:
actually right gramatically speaking all the variations will be wrong cause if got one bidder in the list it will be Listed all bidder which makes no sense. Maybe we should all change to Displaying full bidder list